src/Context/cartContext.js

```diff
@@ -0,0 +1,43 @@
+import { createContext, useContext, useReducer } from "react";
+
+const CartContext = createContext();
+
+const cartFunction = (cartState, action) =>{
+    switch(action.type) {
+        case 'Add_to_cart':
```

Comment:
use of all capital letters is advised here, example case "ADD_TO_CART"

Reply:
Okay will change that.